docs/output.md

```diff
@@ -14,457 +14,415 @@ All paths are relative to the top-level results directory.
-To re-generate `recalibrated BAM` file you have to apply the recalibration table delivered to the `DuplicatesMarked\` folder either using `Sarek` ( [`--step recalibrate`](usage.md#step-recalibrate) ) , or doing this recalibration yourself.
+These files are intermediate and by default not kept in the final files delivered to users. Set `--save_bam_mapped` to enable publishing.
 
-For all samples:
+_For all mappers and samples_:
 
-**Output directory: `results/Preprocessing/[SAMPLE]/Recalibrated`**
+**Output directory: `{outdir}/preprocessing/<sample>/mapped`**
 
-- `[SAMPLE].recal.bam` and `[SAMPLE].recal.bam.bai`
+- `<sample>.bam` and `<sample>.bam.bai`
   - `BAM` file and index
 
-For further reading and documentation see the [data pre-processing for variant discovery from the GATK best practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035535912-Data-pre-processing-for-variant-discovery).
-
-### CSV files
-
-The `CSV` files are auto-generated and can be used by `Sarek` for further processing and/or variant calling.
+### Mark Duplicates
 
-For further reading and documentation see the [`--input`](usage.md#--input) section in the usage documentation.
+During duplicate marking, read pairs that are likely to have originated from duplicates of the same original DNA fragments through some artificial processes are identified. These are considered to be non-independent observations, so all but a single read pair within each set of duplicates are marked, causing the marked pairs to be ignored by default during the variant discovery process
 
-For all samples:
+For further reading and documentation see the [data pre-processing for variant discovery from the GATK best practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035535912-Data-pre-processing-for-variant-discovery).
 
-**Output directory: `results/Preprocessing/TSV`**
+#### GATK MarkDuplicates (Spark)
 
-- `duplicates_marked_no_table.tsv`, `duplicates_marked.tsv` and `recalibrated.tsv`
-  - `TSV` files to start `Sarek` from `prepare_recalibration`, `recalibrate` or `variantcalling` steps.
-- `duplicates_marked_no_table_[SAMPLE].tsv`, `duplicates_marked_[SAMPLE].tsv` and `recalibrated_[SAMPLE].tsv`
-  - `TSV` files to start `Sarek` from `prepare_recalibration`, `recalibrate` or `variantcalling` steps for a specific sample.
+By default, `Sarek` will use [GATK MarkDuplicates](https://gatk.broadinstitute.org/hc/en-us/articles/5358880192027-MarkDuplicates-Picard-).
 
-### CSV files with `--skip_markduplicates`
+Specify `--use_gatk_spark markduplicates` to use [`GATK MarkDuplicatesSpark`](https://gatk.broadinstitute.org/hc/en-us/articles/5358833264411-MarkDuplicatesSpark) instead, the corresponding `Spark` implementation. The resulting files are converted to `CRAM` either with [samtools](https://www.htslib.org/doc/samtools.html) when `GATK MarkDuplicates` is used or implicitly by `GATK MarkDuplicatesSpark`
```

Comment:
```suggestion
Specify `--use_gatk_spark markduplicates` to use [`GATK MarkDuplicatesSpark`](https://gatk.broadinstitute.org/hc/en-us/articles/5358833264411-MarkDuplicatesSpark) instead of the corresponding Spark implementation. The resulting files are converted to `CRAM` with either [samtools](https://www.htslib.org/doc/samtools.html), when `GATK MarkDuplicates` is used, or, implicitly, by `GATK MarkDuplicatesSpark`.
```